probe.go

```diff
@@ -670,6 +670,121 @@ func probeLicenseStatus(c FortiHTTP) ([]prometheus.Metric, bool) {
 	return m, true
 }
 
+func probeLinkMonitor(c FortiHTTP) ([]prometheus.Metric, bool) {
+	var (
+		linkStatus = prometheus.NewDesc(
+			"fortigate_link_status",
+			"signals if the status is up (1) or down (0)",
+			[]string{"vdom", "link_group", "link"}, nil,
+		)
+		linkLatency = prometheus.NewDesc(
+			"fortigate_link_latency_seconds",
+			"latency of this link in seconds",
+			[]string{"vdom", "link_group", "link"}, nil,
+		)
+		linkJitter = prometheus.NewDesc(
+			"fortigate_link_jitter_seconds",
```

Comment:
Given the discussion, WDYT about `fortigate_link_latency_jitter_seconds`?

Reply:
yeah, that makes it clearer

Reply:
Are the averages always for 500ms like your screenshots suggest or is this some setting on the monitor?

Reply:
It is the probe interval it seems. I changed it by `set interval 750` and now it says `Interval: 750 ms`.

The avg is probably based on this:
```
ftg01 (1) # set 
[...]
probe-count                 Number of most recent probes that should be used to calculate latency and jitter (5 - 30, default = 30).
[..]
```